manifests/helm-profiles/demo.yaml

```diff
@@ -7,16 +7,16 @@ meshConfig:
   extensionProviders:
     - name: otel
       envoyOtelAls:
-        service: opentelemetry-collector.istio-system.svc.cluster.local
+        service: opentelemetry-collector.observability.svc.cluster.local
         port: 4317
     - name: skywalking
       skywalking:
-        service: tracing.istio-system.svc.cluster.local
+        service: tracing.observability.svc.cluster.local
```

Comment:
`samples/addons/jaeger.yaml`  hardcodes to istio-system. Is that an issue? I see this is 'skywalking' so maybe not

Reply:
skywalking is hardcode to `istio-system` in [skywalking.yaml](https://github.com/istio/istio/blob/100cb541ab38f8a467226f8865f6f61e581a2a35/samples/addons/extras/skywalking.yaml#L16), reverted and will do it later.